api/base/filters.py

```diff
@@ -148,6 +148,19 @@ def _parse_date_param(self, field, field_name, op, value):
                 'value': stop
             }]
 
+    def bulk_get_values(self, value, field):
+        """
+        Returns list of values from query_param for IN query
+
+        If url contained `/nodes/?filter[id]=12345, abcde`, the returned values would be:
+        [u'12345', u'abcde']
+        """
+        if value[0] == '[' and value[-1] == ']':
+            value = value.replace('[', '').replace(']', '')
```

Comment:
If `value` will never contain additional brackets (and this might already break things if it does), these two lines could be replaced with `value = value.lstrip('[').rstrip(']')`, which would have no effect if the conditions in the above `if` statement weren't met.